pkg/image/importer/importer.go

```diff
@@ -509,6 +510,14 @@ func (isi *ImageStreamImporter) importRepositoryFromDocker(ctx gocontext.Context
 				continue
 			}
 		}
+
+		signatures, err := signature.GetSignatureForImage(repository.Registry.Hostname(), repository.Name, importDigest.Image.Name)
+		if err == nil {
+			importDigest.Image.Signatures = signatures
```

Comment:
actually I think this code will update the image if the image already exists and it will set the signatures based on the latest data, so this is actually ok.